server/views/pages/recommendations/selectIndexOffence.njk

```diff
@@ -143,23 +143,23 @@
                                 Date of sentence
                             </dt>
                             <dd class="govuk-summary-list__value">
-
+                                {{ hyphenForBlank( formatDateTimeFromIsoString({ isoDate: convictionDetail.dateOfSentence }) ) }}
                             </dd>
                         </div>
                         <div class="govuk-summary-list__row">
                             <dt class="govuk-summary-list__key">
                                 Sentence type
                             </dt>
                             <dd class="govuk-summary-list__value">
-
+                                {{ hyphenForBlank(convictionDetail.sentenceDescription) }}
                             </dd>
                         </div>
                         <div class="govuk-summary-list__row">
                             <dt class="govuk-summary-list__key">
                                 Start date
                             </dt>
                             <dd class="govuk-summary-list__value">
-
+                                {{ hyphenForBlank( formatDateTimeFromIsoString({ isoDate: convictionDetail.dateOfSentence }) ) }}
```

Comment:
The story says that Start date should not be included because we don't have a value for it.

Reply:
Thanks for the spot Darren! Corrected.